sql/core/src/test/scala/org/apache/spark/sql/execution/HiveResultSuite.scala

```diff
@@ -44,4 +45,11 @@ class HiveResultSuite extends SparkFunSuite with SharedSQLContext {
     val tpe = new ExamplePointUDT()
     assert(HiveResult.toHiveString((point, tpe)) === "(50.0, 50.0)")
   }
+
+  test("SPARK-26544: toHiveString correctly escape map type") {
+    val m = Map("log_pb" -> """{"impr_id":"20181231"}""", "request_id" -> "001")
```

Comment:
Could you add more test cases to cover at least `\u` and a couple of `\b`..`\t`.

Reply:
I've add one more case to cover special chars like `\b \t \n \r` and `\u`